src/services/refactors/extractSymbol.ts

```diff
@@ -235,6 +235,17 @@ namespace ts.refactor.extractSymbol {
                     break;
                 }
             }
+
+            if (!statements.length) {
+                // https://github.com/Microsoft/TypeScript/issues/20559
+                // Ranges like [|case 1: break;|] will fail to populate `statements` because
+                // they will never find `start` in `start.parent.statements`.
+                // Consider: We could support ranges like [|case 1:|] by refining them to just
+                // the expression.
+                Debug.assert(isCaseClause(start.parent) && span.start < start.parent.expression.end);

```

Comment:
This assert is incorrect.  There are other ways this can arise.  I'm still trying to determine if a sensible assert is possible.  If not, deleting it should suffice - the only downside of returning early is not being able to extract a range.

Reply:
It looks like it can happen with any non-statement child of a block-like node (e.g. the lbrace in a block).  I'll just remove the assert.